test/test-network/systemd-networkd-tests.py

```diff
@@ -7367,7 +7367,6 @@ def test_dhcp_client_ipv6_only(self):
         # The name ipv6-only option may not be supported by older dnsmasq
         # start_dnsmasq('--dhcp-option=option:ipv6-only,300')
         start_dnsmasq('--dhcp-option=108,00:00:02:00',
-                      '--dhcp-option=option6:sip-server,[2600::dd]',
```

Comment:
This was added in the first commit, and then dropped in the fourth commit. Please drop this from the first commit.